resource/cmake/DdsFunctions.cmake

```
@@ -13,29 +13,29 @@
 #                                                                             #
 ###############################################################################
 
-function(connextdds_run_codegen)
-    set(BOOLEANS)
-    set(SINGLE_VALUE_ARGS IDL_FILE LANGUAGE OUTPUT_DIR)
-    set(MULTI_VALUE_ARGS INPUT_PARAMS)
+function(rtigw_connextdds_run_codegen)
+    set(_BOOLEANS)
+    set(_SINGLE_VALUE_ARGS IDL_FILE LANGUAGE OUTPUT_DIR)
+    set(_MULTI_VALUE_ARGS INPUT_PARAMS)
 
     cmake_parse_arguments(_CONNEXTDDS
```

I would try to stick to using `_args` as the prefix for "complex" arguments parsed by `cmake_parse_arguments()`. Beside, `_CONNEXTDDS` is not really a fitting prefix for this function.

```suggestion
    cmake_parse_arguments(_args
```